### PR TITLE
MCS-530 Updates to Support Collisions

### PR DESCRIPTION
**Pull request by ThomasSchellenbergNextCentury**
- Removed AI2-THOR v2.5's default drag
- Added default bounciness to some objects
- Added the `ghosts` JSON property to make objects temporarily intangible
- Made the room bigger in all intuitive physics scenes, because otherwise a moving object may bounce off a wall and back on screen.

### CODE REVIEW

**Review by erick-u — APPROVED** [review]

**Review by kdrumm — APPROVED** [review]
Looks good.  Do we need integration tests in MCS?

**Review by rartiss55 — APPROVED** [review]

**Comment by ThomasSchellenbergNextCentury** [comment]
> Looks good. Do we need integration tests in MCS?

Yes, that was on my list of things to do today, though I should have mentioned it here too. Thanks!

**Comment by ThomasSchellenbergNextCentury** [comment]
I THINK this is done now...

**Review by rartiss55 — APPROVED** [review]

File: unity/Assets/Resources/MCS/Custom/Materials/Azure.mat [hunk 1]
```diff
@@ -0,0 +1,77 @@
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
+--- !u!21 &2100000
+Material:
+  serializedVersion: 6
+  m_ObjectHideFlags: 0
+  m_CorrespondingSourceObject: {fileID: 0}
+  m_PrefabInstance: {fileID: 0}
+  m_PrefabAsset: {fileID: 0}
+  m_Name: Black
```
Comment (rartiss55) [inline unity/Assets/Resources/MCS/Custom/Materials/Azure.mat:10]: @ThomasSchellenbergNextCentury Whoops -- I think I hit approve already but I noticed this after the fact -- I think the color names need to be updated in all the new files (I'm guessing you used "Black" as a template).

Comment (ThomasSchellenbergNextCentury) [reply unity/Assets/Resources/MCS/Custom/Materials/Azure.mat]: Ugh, thanks for pointing this out. Not only was the name incorrect, but also the color! This has happened before... I'm not sure why the stupid Unity Editor doesn't persist my changes to materials. Well, it's fixed, so I can finally go to bed!

**Pull request merged**